javascript_rpc/package.json

```diff
@@ -20,8 +20,13 @@
   "bugs": {
     "url": "https://github.com/EdgePi-Cloud/edgepi-rpc-protobuf/issues"
   },
-  "homepage": "https://github.com/EdgePi-Cloud/edgepi-rpc-protobuf#readme",
+  "homepage": "https://github.com/EdgePi-Cloud/edgepi-rpc-protobuf/tree/main/javascript_rpc#readme",
```

Comment:
would the following work ? make sure to test before changing it.
```
https://github.com/EdgePi-Cloud/edgepi-rpc-protobuf/javascript_rpc
```

Reply:
I need the tree/main part but I can take away #readme.